src/main/kotlin/no/nav/syfo/consumer/narmesteleder/NarmesteLederClient.kt

```diff
@@ -0,0 +1,76 @@
+package no.nav.syfo.consumer.narmesteleder
+
+import no.nav.syfo.consumer.azuread.v2.AzureAdV2TokenConsumer
+import no.nav.syfo.util.*
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.*
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+
+@Service
+class NarmesteLederClient(
```

Comment:
Burde den hete `IsNarmesteLederClient`? Så vi ikke tror det er `narmesteleder`-appen?

Reply:
Jeg ser jo at det kan forveksles med `is` i betydningen "er"(boolean)

Reply:
Ja var litt usikker på det selv, vet ikke om vi bør navngi det som host-applikasjon eller som domene-ord?

Reply:
Ja, usikker jeg og. Hva med å beholde navnet `NarmesteLederClient`, men kalle env variablene  `isnarmesteleder.url` og `isnarmesteleder.client.id`

Reply:
God plan!